From f7953d624a409e748b5998e31345eb15e76f3ad7 Mon Sep 17 00:00:00 2001
From: jkreierh <jkreierh@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 20 Nov 2014 15:26:33 +0000
Subject: [PATCH 0620/1077] MANTIS 34722 Mantis 34722: Report media-low over
 the "Atb2Dev.idl" interface - Extend the mediaStatusType enumeration by
 MEDIA_LOW.

git-svn-id: svn://localhost/SelfServiceCommon/trunk@778 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/atb2dev.idl | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/atb2dev.idl b/SelfServiceCommon/Massai/idl/Massai/atb2dev.idl
index 7577785e..1afa8bc2 100644
--- a/SelfServiceCommon/Massai/idl/Massai/atb2dev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/atb2dev.idl
@@ -76,7 +76,8 @@ module atb2dev
     enum mediaStatusType
     {
       MEDIA_ABSENT,             /**< Media present in device. */
-      MEDIA_PRESENT             /**< Media not present in device. */
+      MEDIA_PRESENT,            /**< Media not present in device. */
+      MEDIA_LOW                 /**< Media in device is nearly gone. */
     };
 
     /** Defines the current status of a media stock. */
-- 
2.41.0.windows.1

